cli/cmd/test_connection.go

```diff
@@ -291,3 +303,26 @@ func testPluginConnection(ctx context.Context, client plugin.PluginClient, spec
 		FailureDescription: resp.FailureDescription,
 	}, nil
 }
+
+// fetchFailedTestResults fetch the failed test results.
+//
+// The function returns any failed test results, or nil if all tests passed. The hackernews plugin is excluded from the
+// failed test results since it was previously used as a test case for the test connection command.
+func fetchFailedTestResults(results []testConnectionResult) (*testConnectionResult, error) {
+	var failedResults []testConnectionResult
+
+	for _, result := range results {
+		if !result.Success && !strings.HasPrefix(result.pluginRef, "hackernews") {
```

Comment:
Should exclude the specific version of `hackernews` and `file` plugins if possible:
`cloudquery/file v4.0.4`
`cloudquery/hackernews v3.0.25`